gatsby-theme-landing-page/src/components/benefits.js

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
+import * as styles from "./benefits.module.css";
+import MarkdownText from "./markdown-text";
+import Link from "./link";
+
+export default function Benefits({ heading, secondaryHeading, content }) {
+  return (
+    <section>
+      <div className={styles.container}>
+        <h2 className={styles.heading}>{heading}</h2>
+        {secondaryHeading && (
+          <h3 className={styles.subhead}>{secondaryHeading}</h3>
+        )}
+        <div className={styles.contentContainer}>
+          {content.map((c) => (
+            <Content key={c.id} {...c} />
```

Comment:
I'd prefer a more semantic name like `Benefit` for this component

Reply:
Can do that; mostly considering this private API since it's scoped to this component and not exported. The `c` var name bothers me more here; should that be something like `item` instead?